egs2/mini_librispeech/diar1/conf/train_diar_eda.yaml

```diff
@@ -0,0 +1,54 @@
+model_conf:
```

Comment:
Please add more information to the models (e.g., related papers). EDA and SA are familiar around us but not for all.

Reply:
Added information in `train_diar_eda.yaml` and `espnet2/diar/espnet_model.py`.

Reply:
Very cool! thanks